en/simulation/README.md

```diff
@@ -155,9 +155,8 @@ make px4_sitl none_iris
 
 The simulation can be further configured via environment variables:
 
-- `PX4_ESTIMATOR`: This variable configures which estimator to use.
-  Possible options are: `ekf2` (default), `lpe` (deprecated).
-  It can be set via `export PX4_ESTIMATOR=lpe` before running the simulation.
+- Any of the [PX4 parameters](../advanced_config/parameter_reference.md) can be overridden via `export PX4_PARAM_{name}={value}`.
```

Comment:
This is cool. Are there any others we know that can be used? I saw that `PX4_GZ_STANDALONE` is being discussed in https://github.com/PX4/PX4-Autopilot/pull/22467

Reply:
I have not seen any, most of these, if not all, are just script parameters. Having an overview of those would be useful as well.

Reply:
OK. Let's leave "as is" then for now. The script parameters for the standalone script will be documented, but not here.